modin/experimental/core/execution/native/implementations/hdk_on_native/expr.py

```diff
@@ -1397,4 +1397,11 @@ def build_dt_expr(dt_operation, col_expr):
 
     res = OpExpr("PG_EXTRACT", [operation, col_expr], _get_dtype("int32"))
 
+    if dt_operation == "isodow":
+        res = res.sub(LiteralExpr(1))
+    elif dt_operation == "microsecond":
+        res = res.mod(LiteralExpr(1000000))
+    elif dt_operation == "nanosecond":
+        res = res.mod(LiteralExpr(1000))
+
```

Comment:
shouldn't it be something like?
```suggestion
    elif dt_operation == "nanosecond":
        res = res.mod(LiteralExpr(1000))
    else:
        raise NotImplementedError(dt_operation)
```

Reply:
No, this piece of code is needed to modify some operations whose behavior differs between pandas and hdk. The behavior of the remaining operations that are currently being transmitted does not need to be modified.

Reply:
@dchigarev Any other comments/questions?